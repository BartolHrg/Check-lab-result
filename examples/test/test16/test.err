java.nio.file.FileSystemException: ProgBroken.jar: The process cannot access the file because it is being used by another process.

	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:92)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileCopy.move(WindowsFileCopy.java:384)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.move(WindowsFileSystemProvider.java:292)
	at java.base/java.nio.file.Files.move(Files.java:1424)
	at jdk.jartool/sun.tools.jar.Main.validateAndClose(Main.java:466)
	at jdk.jartool/sun.tools.jar.Main.run(Main.java:349)
	at jdk.jartool/sun.tools.jar.Main.main(Main.java:1681)
RNGesus decided this should be an error
