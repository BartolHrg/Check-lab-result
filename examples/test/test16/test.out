75
59
536
